sample-input/benchmarks/homogeneous-one-group/HOG-isotope.py

```diff
@@ -0,0 +1,118 @@
+import numpy
+from openmoc import *
+import openmoc.log as log
+import openmoc.plotter as plotter
+from openmoc.options import Options
+
+
+###############################################################################
+#######################   Main Simulation Parameters   ########################
+###############################################################################
+
+options = Options()
+
+num_threads = options.getNumThreads()
+track_spacing = options.getTrackSpacing()
+num_azim = options.getNumAzimAngles()
+tolerance = options.getTolerance()
+max_iters = options.getMaxIterations()
+
+log.set_log_level('NORMAL')
+
+log.py_printf('TITLE', 'Simulating a one group homogeneous infinite medium...')
+log.py_printf('HEADER', 'The reference keff = 1.43...')
+
+
+###############################################################################
+#######################   Main Simulation Parameters   ########################
+###############################################################################
+
+log.py_printf('NORMAL', 'Creating materials...')
+
+iso = Isotope(name='1-group infinite medium isotope')
+iso.setNumEnergyGroups(1)
+iso.setSigmaA(numpy.array([0.069389522]))
+iso.setSigmaF(numpy.array([0.0414198575]))
+iso.setNuSigmaF(numpy.array([0.0994076580]))
+iso.setSigmaS(numpy.array([0.383259177]))
+iso.setChi(numpy.array([1.0]))
+iso.setSigmaT(numpy.array([0.452648699]))
+
+infinite_medium = IsoMaterial(name='1-group infinite medium')
+infinite_medium.addIsotope(iso,1.0)
```

Comment:
you might just want to add a comment clarifying what the ``1.0`` represents - a/o, w/o, etc.